server/repository/src/migrations/v1_06_00/is_sync_update.rs

```diff
@@ -210,12 +210,12 @@ pub(crate) fn migrate(connection: &StorageConnection) -> anyhow::Result<()> {
     )
     .is_err()
     {
-        println!("Database migration: Failed to add is_sync_update column to name, name_store_join, clinician and clinician_store_join");
+        println!("Database migration warning: Failed to add is_sync_update column to name, name_store_join, clinician and clinician_store_join as current version is < 1.2.00");
```

Comment:
The message was too misleading and if there aren't any other errors it makes me think that is_sync_update is the thing that is causing the problem...